README.md

```diff
@@ -1,2 +1,126 @@
+
+[LDM-612](http://ls.st/LDM-612) describes "Plans and Policies for LSST Alert Distribution," including a high-level overview of the relevant components, a summary of data rights issues, and extensive discussion of the role and selection of community alert brokers.
+
+[DMTN-093](https://dmtn-093.lsst.io/) presents the technical design of the alert distribution system.
+
+[DMTN-102](https://dmtn-102.lsst.io/) summarizes "Key Numbers" for alerts, estimating numbers, sizing, etc.
+
+[lsst.alert.packet](https://github.com/lsst/alert_packet) is a Python package that contains alert schemas as well as utilities for reading and writing Rubin alert packets.
+
+Community broker authors will be interested in simulating the Kafka alert stream itself.  [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) provides the relevant code; the overall design is discussed in 
+[DMTN-149](https://dmtn-149.lsst.io/).
+[alert-stream](https://github.com/lsst-dm/alert_stream) currently contains additional code for converting from the [Confluent Wire Format](https://dmtn-093.lsst.io/#management-and-evolution) used in the Kafka stream.
+
+## Data Sources
+
+### DECam-HiTS
+
+Sample alerts are provided from a subset of the DECam High Cadence Transient Search (HiTS; [Förster et al. 2016](https://ui.adsabs.harvard.edu/abs/2016ApJ...832..155F/abstract)).
+
+We process g-band data from three fields (`Blind15A_26`, `Blind15A_40`, and `Blind15A_42`) from the 2015 campaign and use coadded templates built from the matching fields in the 2014 campaign.  
+In total there are 5020 processed CCD-visits.
+
+Users interested in performing their own reductions can obtain these data [here](https://github.com/lsst/ap_verify_hits2015).
+The `ap_verify` package in the LSST Stack can be used to [process this dataset](https://pipelines.lsst.io/modules/lsst.ap.verify/running.html).
+
+## Limitations and Caveats
+
+The Alert Production pipelines are not yet complete, so not all of the alert contents prescribed by the DPDD are populated in these sample alerts.
+Missing fields will have `null` entries.
+We describe the most significant omissions below.
+
+### Sample
+Each `DIASource` detected at or above 5 sigma in the difference image produces a `DIASource` entry in the PPDB and a corresponding alert packet.
+Because detection SNR is different than the measured SNR of a point-source fit, not all `DIASources` will have `psFlux/psFluxErr > 5`.
+
+The sample is not filtered for artifacts or bad subtractions.
+Through visual inspection we suggest that the current proportion of artifacts to real transients is about 4:1.
+We expect continued improvement in this ratio.
+
+### Formats
+The Avro format, schema structure, and alert contents released here reflect the current baseline, but users should be prepared for continued evolution and potentially breaking changes.
+
+The top-level `AlertID` record is currently not populated.
```

Comment:
`alertId` is not a nullable field. What do we put in there?

Some nits on phrasing:

```suggestion
The top-level `alertId` field is currently not populated.
```

Reply:
Currently it's just the DIASource id, but  DM-24858 will change that.  I'll update the README.

https://github.com/lsst/ap_association/blob/87898ef4749329fb0c47ff2995ee90c5b53f7ae5/python/lsst/ap/association/packageAlerts.py#L147